spec/spec_helper.rb

```diff
@@ -1,5 +1,8 @@
 require 'rubygems'
+require 'webmock/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.